TESTING.asciidoc

```diff
@@ -489,6 +483,23 @@ supports a remote debugging option:
 gradle run --debug-jvm
 ---------------------------------------------------------------------------
 
+== Debugging remotely from an IDE
+
+If you want to run elasticsearch and be able to remotely attach the process
+for debugging purposes from your IDE, you need to add the following line in
+`config/jvm.options`:
```

Comment:
Also, this can be done from the command line using `ES_JAVA_OPTS` (I do this all the time).